dev/storage_lookup.h

```diff
@@ -133,5 +133,12 @@ namespace sqlite_orm {
             using table_type = storage_pick_table_t<Lookup, DBOs>;
             return std::get<table_type>(dbObjects);
         }
+
+        template<class Lookup, class DBOs, satisfies<is_db_objects, DBOs> = true>
+        auto lookup_table(const DBOs& dbObjects);
+
+        template<class Lookup, class DBOs, satisfies<is_db_objects, DBOs> = true>
+        std::string lookup_table_name(const DBOs& dbObjects);

```

Comment:
Returning a pointer would make using this function very awkward.
But good that you mention that! We should return a reference to the table name instead, otherwise an empty string.